mpmp-client/src/view.java

```diff
@@ -4,172 +4,159 @@
 import java.awt.event.ComponentAdapter;
 import java.awt.event.ComponentEvent;
-import java.awt.geom.Ellipse2D;
 import java.io.File;
 import java.io.IOException;
 import javax.imageio.ImageIO;
 import javax.swing.BoxLayout;
 import javax.swing.ImageIcon;
 import javax.swing.JButton;
+import javax.swing.JFrame;
 import javax.swing.JLabel;
 import javax.swing.JPanel;
-import org.apache.batik.anim.dom.SVGDOMImplementation;
-import org.apache.batik.svggen.SVGGraphics2D;
+import javax.swing.JTextField;
+import javax.swing.JTextPane;
+import org.apache.batik.anim.dom.SAXSVGDocumentFactory;
 import org.apache.batik.swing.JSVGCanvas;
-import org.w3c.dom.DOMImplementation;
-import org.w3c.dom.Element;
+import org.apache.batik.swing.gvt.GVTTreeRendererAdapter;
+import org.apache.batik.swing.gvt.GVTTreeRendererEvent;
+import org.apache.batik.util.XMLResourceDescriptor;
+import org.w3c.dom.Document;
 import org.w3c.dom.svg.SVGDocument;
 
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
 /**
  * @author Klaus, Oskar
  */
 public class view extends javax.swing.JFrame {
+
     JSVGCanvas canvas;
 
     public view() {
-	initComponents();
-	canvas = new JSVGCanvas();
 	setTitle("Monopoly Multiplayer");
-	canvas.setURI(new File("src/res/gameboard.svg").toURI().toString());
-	
+	setResizable(true);
+	setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+	canvas = new JSVGCanvas();
+	canvas.setBackground(new Color(0, 0, 0, 0));
+
 	try {
 	    setContentPane(new JLabel(new ImageIcon(ImageIO.read(new File("src/res/background.png")))));
 	    canvas.setFont(Font.createFont(Font.TRUETYPE_FONT, this.getClass().getResourceAsStream("/res/SourceSansPro-Light.ttf")));
+
+	    String parser = XMLResourceDescriptor.getXMLParserClassName();
+	    SAXSVGDocumentFactory f = new SAXSVGDocumentFactory(parser);
+	    String uri = new File("src/res/gameboard.svg").toURI().toString();
+	    Document doc = f.createDocument(uri);
+	    canvas.setDocument(doc);
 	} catch (FontFormatException | IOException e) {
+
 	}
-	
-	setLayout(new BorderLayout());
-	Dimension d = getPreferredSize();
-	d.height = d.height - 50;
-	d.width = d.width - 50;
-	canvas.setPreferredSize(d);
-	canvas.setBackground(new Color(0, 0, 0, 0));
-	//SVGDocument svgd = canvas.getSVGDocument();
-	add(canvas);
-	
-	//Background optimieren:
-	//https://tips4java.wordpress.com/2008/10/12/background-panel/
-	//TODO: Herr Hanauska fragen, ob das geht.
 
-	this.addComponentListener(new ComponentAdapter() {
-	    @Override
-	    public void componentResized(ComponentEvent e) {
+	canvas.addGVTTreeRendererListener(new GVTTreeRendererAdapter() {
+	    public void gvtRenderingPrepare(GVTTreeRendererEvent e) {
+		setTitle("Monopoly Multiplayer - Loading...");
+	    }
+
+	    public void gvtRenderingCompleted(GVTTreeRendererEvent e) {
+		setTitle("Monopoly Multiplayer");
```

Comment:
Our title is `mpmp - Monopoly Multiplayer`. In the future, we should probably avoid the `Monopoly` entirely, as it is a registered trademark.